routers/chats/apis.py

```
@@ -0,0 +1,421 @@
+
+@chats_router.post('/profile')
+@x_super_team
+@x_app_key
+@jwt_token
+async def profile(request: Request):
+    try:
+        data = await request.form()
+
+        required_fields = ['username', 'token', 'queue']
+        if not check_required_fields(data, required_fields):
+            raise HTTPException(status_code = 400, detail = f"An error occurred: missing parameter(s)")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)